### DISCUSSION TITLE
De-duplication without interpolation

### DISCUSSION THREAD

**Discussion by themattypower** [n00mkrad/flowframes#95] [Ideas]
Hello, could you add a '1X' option to the 'Output FPS and Speed' drop down menu.
I would like to lower the frame rate and simply removing duplicate frames, for example from (progressive) 30fps to 24fps.
If that is even practically possible to do.
Or change a 30fps with duplicates to lower but variable frame rate.
Thank you

**Comment by mirh** [comment]
Decimation is the name of what you seem to want (which indeed flowframes does thorugh imagemagick AFAICT), see also the operation of reverse telecine for interlaced video. 
But putting aside I'm not exactly sure the algorithms can work in reverse (also, it's hard to see why one would want to destroy information) there's no point in using this program if you don't want interpolation.
